### PR TITLE
Invoices downloadable at shop's order view

### PR DESCRIPTION
**Pull request by bartoszpietrzak1994**
screen shot 2018-07-09 at 15 16 54

```
Scenario: Downloading an invoice on order view
    Given I view the summary of the order "#00000666"
    When I click on first invoice's download button
    Then the pdf file for this invoice should be downloaded successfully
```

Scenario fulfilled

### CODE REVIEW

File: features/downloading_invoices_on_customer_order_view.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        And the store allows paying with "Cash on Delivery"
+        And I am a logged in customer
+        And I placed an order "#00000666"
+        And I bought a single "Angel T-Shirt"
+        And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+        And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
+        And I chose "Free" shipping method with "Cash on Delivery" payment
+
+    Scenario: Downloading an invoice on order view
+        Given I view the summary of the order "#00000666"
```
Comment (GSadee) [inline features/downloading_invoices_on_customer_order_view.feature]: When

File: tests/Behat/Page/Shop/Order/ShowPage.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Behat\Page\Shop\Order;
+
+use Sylius\Behat\Page\SymfonyPage;
+use Behat\Mink\Element\NodeElement;
+
+final class ShowPage extends SymfonyPage implements ShowPageInterface
+{
+    public function getRouteName()
```
Comment (GSadee) [inline tests/Behat/Page/Shop/Order/ShowPage.php]: Missing return type

File: tests/Behat/Page/Shop/Order/ShowPageInterface.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Behat\Page\Shop\Order;
+
+interface ShowPageInterface
+{
+    public function downloadFirstInvoice(): void;
+    public function isPdfFileDownloaded(): bool;
```
Comment (GSadee) [inline tests/Behat/Page/Shop/Order/ShowPageInterface.php]: Missing blank line before this line

File: tests/Behat/Resources/suites/suites.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+imports:
+    - admin_suites.yml
+    - customer_suites.yml
```
Comment (GSadee) [inline tests/Behat/Resources/suites/suites.yml]: `admin_suites` => `admin`
`customer_suites ` => `customer`

File: tests/Application/app/config/routing.yml [hunk 5]
```diff
@@ -1,8 +1,12 @@
 sylius:
     resource: "../../../../vendor/sylius/sylius/app/config/routing.yml"
 
-# Put your own routes here
-
-sylius_invoicing_plugin:
-    resource: "@SyliusInvoicingPlugin/Resources/config/app/routing.yml"
+sylius_invoicing_plugin_admin:
+    resource: "@SyliusInvoicingPlugin/Resources/config/app/routing/admin_invoicing.yml"
```
Comment (pamil) [inline tests/Application/app/config/routing.yml]: Let's follow the convention we use for core bundles and PluginSkeleton and put routing in `@SyliusInvoicingPlugin/Resources/config/admin_routing.yml` and `@SyliusInvoicingPlugin/Resources/config/shop_routing.yml` respectively.

**Comment by Zales0123** [comment]
Thank you Bartoszek! 🔭

**Pull request merged**